templates/channels/types/infobip_ussd/claim.haml

```diff
@@ -0,0 +1,55 @@
+-extends "smartmin/form.html"
+-load i18n
+
+-block extra-script
+  {{ block.super }}
+
+  :javascript
+    $(function(){
+      $("#id_country").select2();
+    });
+
+-block title
+  .medium-help.icon-phone.float-left
```

Comment:
All of the <type>/claim.html should have the same header that takes the icon and title from the type class